spigot/src/main/java/net/apartium/cocoabeans/spigot/inventory/ItemBuilder.java

```
@@ -82,24 +63,30 @@ public ItemBuilder setSkullTextureBase64(String base64) {
 
         int start = decodeText.indexOf("http"), end = decodeText.indexOf("\"}}}");
         if (start == -1 || end == -1) return this;
-        setSkullTextureURL(decodeText.substring(start, end));
+        try {
+            setSkullTextureURL(new URL(decodeText.substring(start, end)));
+        } catch (MalformedURLException e) {
+            return this;
```

Throw an exception here, silently failing especially here is really bad